charts/presto/templates/hive-server-statefulset.yaml

```diff
@@ -80,6 +80,10 @@ spec:
           mountPath: /hadoop/dfs/name
         - name: datanode-empty
           mountPath: /hadoop/dfs/data
+{{- if and .Values.spec.config.sharedVolume.enabled .Values.spec.config.sharedVolume.createPVC }}
+        - name: hive-warehouse-data
+{{ toYaml .Values.spec.config.sharedVolume.mountPath | indent 10 }}
```

Comment:
This doesn't produce valid yaml. You should just specify `path:` here directly and just directly template `.Values.spec.config.sharedVolume.mountPath` (without toYaml`) as the value.